SimplenoteUITests/SimplenoteUITestsNoteEditor.swift

```diff
@@ -312,4 +312,49 @@ class SimplenoteUISmokeTestsNoteEditor: XCTestCase {
         TrashAssert.noteExists(noteName: noteTitle + "-1")
         TrashAssert.noteExists(noteName: noteTitle + "-2")
     }
+
+    func testUndoTrashFromSnackbar() throws {
+        trackTest()
+        let noteName = "Note Title"
+
+        trackStep()
+        populateNoteList(title: noteName, numberToCreate: 1)
+
+        trackStep()
+        NoteList.longPressNote(title: noteName + "-1")
+        NoteList.deleteNoteFromContextMenu()
+        NoteListAssert.notesNumber(expectedNotesNumber: 0)
+
+        trackStep()
+        NoteList.tapUndoOnSnackbar()
+        NoteListAssert.notesNumber(expectedNotesNumber: 1)
+        NoteListAssert.noteExists(noteName: noteName + "-1")
+    }
+
+    func testCopyAndPasteInterlink() throws {
+        trackTest()
+        let toCopyNoteTitle = "Copy Internal Link Title"
+        let toPasteNoteTitle = "Paste Internal Link Title\n"
+        let note = NoteData(
+            name: "Paste Internal Link Title",
+            content: "[Copy Internal Link Title-1](simplenote://note/"
```

Comment:
a lot of strings here and not sure how to make this one cleaner, they're similar but not entirely the same.